docs/orcfax-feeds/feed-overview.md

```diff
@@ -53,77 +35,144 @@ For a feed of type CER, the naming convention is
 
 :::
 
-[cer-1]: https://glossary.orcfax.io/#cer
-[cer-2]: https://glossary.orcfax.io/#baseQuote
-
-### Feeds derived from CEXes
+The price values published by these feeds are derived from centralized exchange
+([CEX][cex-1]) data or decentralized exchange ([DEX][dex-1]) data.
 
-The price values published by these feeds are derived entirely from centralized
-exchange ([CEX][cex-1]) data.
+For CEX based feeds, fiat currency names follow ISO [4217][cex-2] and ordering
+follows [forex][cex-3] conventions where applicable. In other cases, such as for
+crypto base currencies, ordering defers to how the pairs appear on CEXes.
 
-Fiat currency names follow ISO [4217][cex-2] and Ordering follows [forex][cex-3]
-convention where applicable. In other cases, such as for crypto base currencies,
-ordering defers to how the pairs appear on CEXes.
+For DEX based feeds, ordering defers to how pairs appear on DEXes.
```

Comment:
These two sentences are difficult to parse while reading, I wonder if we can simply say:

> In all other cases, ordering defers to how pairs appear on the centralized or decentralized exchanges (CEX/DEX).

I see you've linked to the glossary, an explicit info box might help readers and promote it nicely:

> INFO: The [Glossary](https://glossary.orcfax.io/) provides more information on the terminology and standard adopted by Orcfax.